.golangci.yml

```diff
@@ -1,5 +1,6 @@
 linters:
   enable:
+    - unparam
     - unconvert
     - errorlint
```

Comment:
We should sort this list; helps preventing accidental duplicates and makes it easier to find which ones are enabled

Reply:
done